mcc/odin/odin/mock_rds_client_test.go

```diff
@@ -16,6 +16,30 @@ type mockRDSClient struct {
 	dbSnapshots          []*rds.DBSnapshot
 }
 
+// AddSnapshots add a list of snapshots to the mock, both in the full
+// list and in the per instance map.
+func (m *mockRDSClient) AddSnapshots(
+	snapshots []*rds.DBSnapshot,
+) {
+	m.dbSnapshots = []*rds.DBSnapshot{}
+	for _, snapshot := range snapshots {
+		m.AddSnapshot(snapshot)
+	}
+}
+
+// AddSnapshot add a new snapshot to the mock, both in the full list
+// and the in the per instance map.
+func (m *mockRDSClient) AddSnapshot(
+	snapshot *rds.DBSnapshot,
```

Comment:
I'm afraid I still don't understand the Go method definitions.. What's `m` here and what's `snapshot` – are both method parameters (b.c. both are used in the method's body..)? Or is this some other magic??